### ISSUE TITLE
"Error in processing request" red box on start.

### ISSUE THREAD

**Issue by ethaniel** [phpmyadmin/phpmyadmin#13092]
### Steps to reproduce
1. Open main page.

### Expected behaviour
No errors.

### Actual behaviour
![image](https://cloud.githubusercontent.com/assets/941973/24052253/6bcd5318-0b67-11e7-981c-ca382799f90e.png)

My debugger shows a 500 error when a POST request is sent to /db_structure.php?ajax_request=1&favorite_table=1&sync_favorite_tables=1
![image](https://cloud.githubusercontent.com/assets/941973/24052323/af936bfa-0b67-11e7-9281-e30e671b2ab1.png)

What's weird is when I do a normal GET request to that URL, I receive a response (and not a 500 error).

My php error logs are empty.

Any ideas?

### Server configuration
**Operating system**:
Centos 6.7

**Web server:**
nginx

**Database:**
percona 

**PHP version:**
5.6.30

**phpMyAdmin version:**
4.7.0-rc1

### Client configuration
**Browser:**
Chrome

**Operating system:**
Mac OSX

**Comment by ethaniel** [comment]
The problem went away when I installed the phpmyadmin storage database. But still, this error is confusing and I think it should not appear on a fresh installation.

**Comment by ibennetch** [comment]
I tried and was unable to reproduce this, but I have no thoughts about why
it happened for you.

**Comment by nijel** [comment]
Can you please look to web server error logs to see what has caused the error?

**Comment by nijel** [comment]
I *probably* found where the problem is, but still having that confirmed would be nice...

**Comment by ethaniel** [comment]
Looks like phpMyAdmin has taken over the logging. I don't see any errors in my syslog (which php is configured to log by default).

**Comment by nijel** [comment]
phpMyAdmin does not configure this, so it's most likely the PHP is not configured to do so (eg. if using fpm, you need to enable catch_workers_output to get useful logging from PHP).

**Comment by ethaniel** [comment]
@nijel I added catch_workers_output to fpm, and it started giving me extra logs from my other systems. But none from phpMyAdmin.

I just upgraded to 4.7.0 and still see the dreaded error. Sometimes I get a "token mismatch" error.

I suppose that 500 Internal Server Error is coming from phpMyAdmin itself and not fpm.

Is it possible?

**Comment by nijel** [comment]
No, phpMyAdmin AFAIK does not return 500 on it's own, so it has to come from PHP or web server...

**Comment by ethaniel** [comment]
@nijel guess what? phpMyAdmin does return 500 errors on it's own.

I managed to debug the XHR request and here's what I got:

```
<h1>phpMyAdmin - Error</h1>
<p>Error during session start; please check your PHP and/or webserver log file and configure your PHP installation properly. Also ensure that cookies are enabled in your browser.<br /><br />session_start(): Failed to decode session object. Session has been destroyed</p>
```

**Comment by ethaniel** [comment]
@nijel that means that phpMyAdmin takes over the errors and they don't hit my regular logging system. And the biggest problem is that if an error happens during an ajax request, there is no way of finding out which error it is without using special tools to check the reply.

I think this is a major problem that needs to be looked into.

**Comment by nijel** [comment]
Indeed it should properly report json in this case.

**Comment by nijel** [comment]
This seems to be caused by fix for #12810...
